packages/react-app/src/components/ProfitPie/profit-pie.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Pie } from 'react-chartjs-2';
+import {
+  Chart,
+  ArcElement,
+  PieController,
+  Decimation,
+  Tooltip,
+  Legend,
+} from 'chart.js';
+
+Chart.register(
+  ArcElement,
+  PieController,
+  Decimation,
+  Tooltip,
+  Legend,
+);
+
+function toPercentage(ethValue) {
+  // smart contract can't hold float so I have to divide by 10
+  return ethValue / 10;
+}
+
+function ProfitPie({ freaks }) {
+  const labels = freaks.map(({ name }) => name);
+  const values = freaks.map(({ share }) => toPercentage(share));
+
+  function dynamicColors() {
```

Comment:
95% (made up percentage) of the cases functions should start with a verb

```suggestion
  function getDynamicColors() {
```